src/flag.rs

```diff
@@ -187,7 +203,10 @@ mod tests {
     fn register_unregister() {
         // When we register the action, it is active.
         let flag = Arc::new(AtomicBool::new(false));
+        #[cfg(not(windows))]
         let signal = register(::SIGUSR1, Arc::clone(&flag)).unwrap();
+        #[cfg(windows)]
+        let signal = register(::SIGTERM, Arc::clone(&flag)).unwrap();
```

Comment:
I believe these tests were using USR1 simply to leave SIGTERM alone, so I could CTRL+C the tests if the got into an infinite loop or something. Killing them with SIGKILL works too and is probably better than having these windows/non-windows conditions, so let's use SIGTERM in both.